CHANGELOG.md

```diff
@@ -48,6 +50,8 @@
   [#358](https://github.com/nextcloud/cookbook/pull/358) @christianlupus
 - Avoid recipe are no longer reachable when user changes locales
   [#371](https://github.com/nextcloud/cookbook/pull/371) @christianlupus
+- Creating new recipe not possible due to null reference
+  [#378](https://github.com/nextcloud/cookbook/pull/378/) @seyfeb
```

Comment:
I think this is a leftover from another PR....

Reply:
Yeah, I merged #378 in there to be able to create new recipes at all. I can remove those lines.

Reply:
So, #378 should be merged before. All right.